includes/MediaWiki/Api/EditEndpoint.php

```diff
@@ -184,19 +190,59 @@ public function run() {
 
 		// And make the edit
 		$toSave = $this->simplePutStrategy->apply( $reconciliationItem->getItem(), $inputEntity );
+
 		$editEntity = $this->editEntityFactory->newEditEntity(
```

Comment:
It probably makes sense to move this down, since we don’t use this `$editEntity` until we’ve processed the `$otherItems`.

Reply:
sounds good